orange-tree.rb

```diff
@@ -0,0 +1,132 @@
+# this one is another group project! three adies to a group. :)
+# https://github.com/Ada-Developers-Academy/daily-curriculum/blob/master/topic_resources/orange_tree.md
+# -------------------------------------
+# actually, according to the new dinner requirements, I think we can move on to dessert!
+
+class OrangeTree
+
+  attr_reader :age, :alive, :grove, :height, :orange_count
+
+  def initialize
+    @age = 0
+    @alive = true
+    @height = 0
+    @orange_count = 0
+  end
+
+  def grow_oranges
+    if @age >= 3 # oranges don't grow on young trees!
+      if @grove.soil_quality == "unhealthy"
```

Comment:
That's it exactly. I'm pretty sure that's also how Alice had intended to solve the problem. We hadn't had any time left as a group to work on it, so I'd just turned it in as is. I'm currently updating the code to keep soil quality in the trees & have the grove check & update that.

The way I'm implementing it, solo trees will always have healthy soil quality, because only in crowded groves will they start affecting each other. Then in `one_year_passes`, the grove will check the trees & update the soil quality for them. :)